src/chains/index.ts

```diff
@@ -235,7 +235,8 @@ export { wanchain } from './definitions/wanchain.js'
 export { wanchainTestnet } from './definitions/wanchainTestnet.js'
 export { wemix } from './definitions/wemix.js'
 export { wemixTestnet } from './definitions/wemixTestnet.js'
-export { x1Testnet } from './definitions/x1Testnet.js'
+export { xLayerTestnet, x1Testnet } from './definitions/xLayerTestnet.js'
+export { xLayer, x1 } from './definitions/xLayerTestnet.js'
```

Comment:
```suggestion
export { xLayer } from './definitions/xLayerTestnet.js'
```

Ideally should export one chain (unless deprecated).